api-reference/10 UI Widgets/dxDiagram/1 Configuration/propertiesPanel/tabs/commands.md

```diff
@@ -5,7 +5,7 @@ type: Array<dxDiagramCustomCommand> | Array<String>
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Lists commands in a tab.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+You can populate a tab with commands or with groups of commands. When the [groups](/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/propertiesPanel/tabs/groups/) option is specified, the **commands** option is not in effect.
```

Comment:
What is the difference between 'commands or groups of commands'?

Reply:
Perhaps:
Use the  [groups] option to group commands in the tab. Note, if the [groups] option is specified, the **commands** option is not in effect.

Reply:
Ok